include/aspect/material_model/visco_plastic.h

```diff
@@ -114,7 +114,9 @@ namespace aspect
         void
         parse_parameters (ParameterHandler &prm);
 
-      private:
+        double get_min_strain_rate() const;
```

Comment:
Do you need this function to be `public`? If you only need it in your derived material model it should be sufficient to make it `protected`. Only if you want to use it from particle properties or postprocessors it needs to be public.